.github/workflows/ql-for-ql-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Run QL for QL Tests
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - ql/*
+  pull_request:
+    branches: [main]
+    paths:
+      - ql/*
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  qltest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch CodeQL
```

Comment:
As above:

> See https://github.com/github/codeql/pull/7410/files#diff-c3c811f2f3a821590acf45fc281ba009b4886ec06d156f573019acf671e2d817R23 for how to obtain the codeql-cli that the codeql-action would be using. It is likely present on disk already, and it will by construction be the latest stable release.